libs/core/algorithms/include/hpx/parallel/algorithms/make_heap.hpp

```diff
@@ -293,6 +293,33 @@ namespace hpx::parallel {
             }
         }
 
+        template <typename RndIter, typename Comp, typename Proj>
+        constexpr void sift_down_range(RndIter first, Comp&& comp, Proj&& proj,
+            typename std::iterator_traits<RndIter>::difference_type len,
+            RndIter start, std::size_t count,
+            /*is_unsequenced_policy*/ std::false_type)
+        {
+            for (std::size_t i = 0; i != count; ++i)
+            {
+                sift_down(first, comp, proj, len, start - i);
+            }
+        }
+
+        template <typename RndIter, typename Comp, typename Proj>
+        constexpr void sift_down_range(RndIter first, Comp&& comp, Proj&& proj,
+            typename std::iterator_traits<RndIter>::difference_type len,
+            RndIter start, std::size_t count,
+            /*is_unsequenced_policy*/ std::true_type)
+        {
+            // clang-format off
+            HPX_IVDEP HPX_UNROLL HPX_VECTORIZE
+            for (std::size_t i = 0; i != count; ++i)
+            {
+                sift_down(first, comp, proj, len, start - i);
+            }
+            // clang-format on
+        }
+
         template <typename Iter, typename Sent, typename Comp, typename Proj>
         constexpr Iter sequential_make_heap(
```

Comment:
Same here, is this still needed?

Reply:
I don't think so, the only place where sequential_make_heap() is called has been overloaded. We can delete the original implementation.